### ISSUE TITLE
Suitable for no model organisms? 

### ISSUE THREAD

**Issue by kamalmdmostafa** [jeffersonfparil/compare_genomes#13]
Hi, I still need to go through the details. I just installed and started the Test run. We have sequenced and assembled some ciliate genomes that have never been reported. I am a newbie in the field of evolutionary genomics. Can this pipeline be adapted to those non-model organisms? 

I really appreciate any help you can provide.

**Comment by jeffersonfparil** [comment]
Yes, that should still be possible, but you may need to do the annotations of all the genomes included if the resources are scant. And Paramecium is one of the species with a PantherHMM database. You can have a look at the [list](http://data.pantherdb.org/ftp/sequence_classifications/18.0/PANTHER_Sequence_Classification_files/) for the most closely related species you may want to use.

**Comment by kamalmdmostafa** [comment]
Thank you for getting back to me so quickly. I am currently working with green Tetrahymena, which we have sequenced and annotated. My plan is to compare it with all of the other reported full genome sequences of Tetrahymena species from the [TGD](https://tet.ciliate.org/downloads.php) database. I am wondering if it is necessary to use the pather_HMM database for this pipeline. My ultimate goal is to study gene expansion/contraction and analyze Ka/Ks.

**Comment by jeffersonfparil** [comment]
No worries. That's also possible if you want to classify the orthogroups (output of OrthoFinder) yourself. You may work on the output of OrthoFinder directly if you wish and perform CAFE expansion/contraction analyses on them with or without classifying the orthogroups (i.e. attaching biological function or most likely gene families to each orthogroup.
